lib/src/sai_redis_hash.cpp

```diff
@@ -2,7 +2,46 @@
 
 REDIS_GENERIC_QUAD(HASH,hash);
 
```

Comment:
add REDIS_GENERIC_QUAD(FINE_GRAINED_HASH,fine_grained_hash);

do the same for VS

Reply:
Done